frontend/src/components/dialog/DialogForm.vue

```diff
@@ -139,11 +111,14 @@ export default {
   methods: {
     async doSubmit() {
```

Comment:
There is still quite some code overlap between DialogForm and DialogBottomSheet. I reckon you don't want to put this into DialogUiBase, because DialogUiBase is also used by DetailPane.

Should we put this code overlap in a separate mixin?

Reply:
#3497